### ISSUE TITLE
Naked mode

### ISSUE THREAD

**Issue by jmduke** [buttondown/roadmap#2718]
## TL;DR

`Naked mode` will be a _third_ editor mode, alongside `Fancy mode` and `Markdown mode`. Authors who use `Naked mode` get _fully_ exposed access to the HTML for their email. The existing `custom_email_template` setting will get _unshipped_ in favor of this mode.

## Why?

It should be as easy as possible for someone to bring their own markup and completely redesign their emails from scratch, if so they wish. Right now, the path for doing this is that we give them access to the raw template via the hidden `custom-email-template` setting, which applies Newsletter-wide. This gets the job done, but is a pain to interact with and has a _terrible_ UX.

Instead, we'll offer this option: an author can hop into Naked Mode and copy and paste their HTML from another tool, tweak specific things as they wish, etc.

## Breakdown

TBD, but:

- [ ] Add backend support for `EmailTemplate.naked`
- [ ] Draw the rest of the owl
- [ ] Add docs page
- [ ] Add changelog entry

## Notes

- From an internal plumbing standpoint, we should materialize an email being naked or not on `email.template` — that is to say, the process of entering "naked mode" is, unlike switching between Fancy + Markdown, _changing the template of the email_. (This means a good first step for beginning this work is to add backend support for "Naked mode" as a template.)
- Our goal is _not_ to expand the scope of WYSIWYG or anything like that. From a user-facing experience standpoint, Naked Mode should be an `HTMLInput` — we'll validate tags to the best of our ability, but that's it.

## Open questions / decisions

- [ ] Should we actually call it Naked Mode? (Probably not.)

**Comment by chriskrycho** [comment]
Okay, you teased this in the update email for May, so I came looking (and I imagine others will too)… so what *is* it?!? 😂

**Comment by jmduke** [comment]
@chriskrycho Okay, okay, I wrote out a little bit of elaboration ;)
